apps/backend/src/db/migrations/20250723031408_add_groups_and_permissions_tables.ts

```diff
@@ -0,0 +1,93 @@
+import { type Knex } from "knex";
+
+const TableNames = {
```

Comment:
enums should be named in singular form
```suggestion
const TableName = {
```